meson.build

```diff
@@ -1,13 +1,25 @@
-project('live-chart', ['vala', 'c'], version: '1.9.1')
+project('live-chart', ['vala', 'c'], version: '1.9.1-RO')
 
 cc = meson.get_compiler('c')
 libm = cc.find_library('m', required: true)
 
-gtk = dependency('gtk+-3.0', version: '>= 3.22')
+#setting up GEE
 gee = dependency('gee-0.8')
-
 vala_args = ['--target-glib=2.50']
 
+#setting up gtk
+gtk_major = 3
```

Comment:
GTK version should define in meson_options.txt:
```
option('GTK',
       type: 'integer',
       value: 4,
       description : 'GTK version')
```

then use it in the meson:
```
if 3 == get_option('GTK')
else
endif
```

then change the GTK version by command:
`meson -DGTK=3 build`

Reply:
Thank you.
I tried it, and it works.
I'll take it in my repository later.

Reply:
By the way, as talked in https://github.com/lcallarec/live-chart/pull/37, the default build should be GTK3 for now if we don't provide any argument to meson.